spec/domain/mail_relay/address_list_spec.rb

```diff
@@ -80,6 +80,16 @@
     ])
   end
 
+  it 'works for new records' do
+    person = Fabricate(:person)
+    e1 = Fabricate(:additional_email, contactable: person, mailings: true)
+    Fabricate(:additional_email, contactable: bottom_member, mailings: false)
```

Comment:
Ich dachte Fabricate persistet doch die Person direkt in die DB, oder habe ich das falsch im Kopf? Wenn ich recht habe dann testest du hier gar nicht mit einem new record